app/views/research_sessions/preview.html.erb

```diff
@@ -46,105 +46,84 @@
     </p>
   </section>
 
+  <% methodologies = capture do %>
+    <ul class="bullet-point-list">
+      <% @research_session.methodologies.each do |methodology| %>
+        <li>
+          <%= edit_link_for(:methodologies) do %>
+            <%= methodology_lookup(methodology) %>
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <% recording = capture do %>
+    <p>
+      Which we will record using:
+      <ul class="bullet-point-list">
+        <% @research_session.recording_methods.each do |recording_method| %>
+          <li>
+            <%= edit_link_for(:recording_methods) do %>
+              <%= recording_method_lookup(recording_method) %>
+            <% end %>
+          </li>
+        <% end %>
+      </ul>
+    </p>
+  <% end %>
+
   <%=
     react_component(
-      'ResearcherPreviews/WhoIsDoingTheResearch',
-      component_params(
-        :researcher_job_title, :researcher_name, :researcher_other_name,
-        final_preview: true
-      ),
+      'Shared/WhatHappensInThisResearchSession',
+      component_params(:researcher_job_title, :researcher_name, final_preview: true).merge({
+        methodologies_markup: methodologies,
+        recording_markup: recording,
+        you_or_your_child: you_or_your_child
+      }),
       {
         prerender: true
       }
     )
   %>
 
   <section>
-    <h3 class="subtitle-small" id="what">What will happen next?</h3>
-
-    <% @research_session.methodologies.each do |methodology| %>
+    <h3 class="subtitle-small" id="session-details">Session details</h3>
```

Comment:
If where/when is not enabled, we should not display the Session Details header (as it is followed by blank and then a next header)

Reply:
This is slightly more complex – we've had to upgrade the logic around this section as there are three inter-relating parts that are mixed up together.